<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changestop</class>
 <widget class="QWidget" name="changestop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>356</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>534</width>
    <height>356</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>534</width>
    <height>356</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#changestop{image: url(:/new/prefix1/bus.jpg);}</string>
  </property>
  <widget class="QLineEdit" name="idEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入您想要修改的站台ID：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择您想要修改的字段：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="nameButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>95</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>站台名称</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="numButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>95</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>站台编号</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="locationButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>95</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>站台位置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入您想要修改后的内容：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="contentEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 237, 255);</string>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 237, 255);</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
